### ISSUE TITLE
wg-outreach Meeting 2019-06-12 (ACS 2019 planning kickoff)

### ISSUE THREAD

**Issue by pbakaus** [ampproject/wg-outreach#6]
This is an irregular wg-outreach meeting spearheaded by @mrjoro to kickoff the Contributor Summit 2019 planning. In the future, we'll either have more independent meetings or fold into the weekly wg-outreach meetings.

**Meeting date/time (UTC)**
19:00-20:00 UTC (noon-1pm Pacific)

**Instructions for joining the meeting**
Join the meeting on Google Meet via [this link](meet.google.com/uys-rdfu-rcf).

**Agenda**
*Note: Add a comment on this issue to propose items for the agenda.*

**Comment by mrjoro** [comment]
Proposed agenda:
- Introductions
- Review goals for AMP Contributor Summit
- Current status of ACS planning
- Discuss the best approach for organizing ourselves
- High-level overview of initial task list based on last year's ACS

**Comment by mrjoro** [comment]
**Notes**

- Goals.  We discussed the goals of the ACS to inform the rest of the planning:
  - Encourage more contributions.
  - Continue momentum in building a sense of community in the AMP open source project.
  - Spread knowledge of current best practices (e.g. testing, best practices on the web that apply to AMP).
  - Engage all contributors at a technical level.
  - Solidify governance changes, including TSC/AC/WG face-to-face meetings.
  - Ensure the conference is inclusive.
- Current planning state
  - Google NYC as a venue backup option, Matt looking into other options
  - event planning system (registration/website/etc.)
    - we used a Google internal event planning system last year, looking into alternatives
    - meetup.com? Pratik has a NY meetup group for AMP [ https://www.meetup.com/The-New-York-AMP-Accelerated-Mobile-Pages-Meet-Up-Group/ ] 
    - need to consider data handling for these; could potentially separate out registration piece from the rest of the planning tasks/etc.
- Agenda
  - New Contributor Day (NCD) seems like a good thing to keep; people found it valuable.
  - Although some of the content is evergreen and we could just share videos from last year, meeting other new and experienced contributors has a lot of benefits.
  - We should consider adding more of a “hackathon” flavor to the day.  We could have people do some of the more basic setup / watch the more basic videos beforehand to allow more time for this.
  - We should consider having talks and breakouts on each day (rather than one day of talks and one day of breakouts).  Feedback from attendees last year was that this would be less “information overload” and allow for discussing the topics brought up in the talks more immediately.
  - The Agenda subteams should discuss how to best incorporate TSC/AC/WGs into the agenda.  (Meetings during breakouts?  Or on NCD?)
- Next steps
  - @mrjoro/@pbakaus/Matt will update task list with effort (T-shirt sizes)
  - @mrjoro will update our notes doc with proposed “organizing subteams” (e.g. registration, talks, etc.), then share it for feedback.  After that’s done people can sign up for the subteams that work best for them, and subteams can organize themselves.

**Comment by cathibosco** [comment]
**Good plan:** "We should consider having talks and breakouts on each day (rather than one day of talks and one day of breakouts). Feedback from attendees last year was that this would be less “information overload” and allow for discussing the topics brought up in the talks more immediately."

**Comment by cathibosco** [comment]
I can likely help with promoting the event - creating open graph imagery to go with the website for sharing links to the site out socially and on Slack... design badges, design promotional speaker cards etc... I plan to be there for the conference and would gladly help at registration if needed. I have a lot of visual assets from working on the WodPress plugin at XWP with the teams... 🚀

**Comment by mrjoro** [comment]
Thanks @cathibosco!

Following up on my tasks:
- I proposed 3 organizing subteams--Attendance, Content and Conference Logistics.  Details are in the [notes doc](https://docs.google.com/document/d/10Aqe8uMCDZctCCK993JBS6DRNTxlHgenCVydBRxhv70/edit#heading=h.wm6qfnn832qy).
- I updated the [task list](https://docs.google.com/spreadsheets/d/1ZQIeE02HyQDFlfXPbr1tnq0JLWM0wibGqIdr3Ef_V_Q/edit#gid=0) with rough estimates of effort for each of the tasks.  I also went ahead and put in draft deadlines for many of the tasks--especially those with an effect on attendees/applicants and cross-subteam dependencies.
